OasisDEX/oasis-borrow
# [7230] aave multiply position info

# [aave position info](https://app.shortcut.com/oazo-apps/story/7230/update-position-info-contents-with-actual-content)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
- updates aave multiply position info to the correct tab
  
## How to test 🧪
- check open and manage on aave multiply and earn
- ensure copy is correct

## Todo:
- KB learn more URL for the strategy
- smart defi account URL
